### DISCUSSION TITLE
Scan: request for reducing default payloads for slack

### DISCUSSION THREAD

**Discussion by ha1f** [fastlane/fastlane#17853] [Ideas] [answered]
## Problem

[scan](https://docs.fastlane.tools/actions/scan/) has option to send to slack.

The document says `Beautiful slack notifications of the test results` and the example looks beautiful.

![slack notification example](https://docs.fastlane.tools/img/actions/slack.png)

However, we cannot send with this style because the notification will contain default payloads as below.

<img width="593" alt="スクリーンショット 2020-12-24 22 02 59" src="https://user-images.githubusercontent.com/10028587/103090297-06b98f80-4634-11eb-9762-aa9d3ee2a02a.png">

I don't think this is so beautiful, and I want it to be simpler.

## Solution

In order to reduce unneeded fields, we have to pass empty array for default_payloads of [slack](https://docs.fastlane.tools/actions/slack/) command.

The related part seems to be these.

https://github.com/fastlane/fastlane/blob/master/scan/lib/scan/slack_poster.rb#L46-L57

How about passing `default_payloads: []` by default here, or adding option to disable default_payloads?

**Comment by rogerluan** [comment, marked as answer]
Hi @ha1f 👋 

That makes total sense! I opened a PR to fix that issue: https://github.com/fastlane/fastlane/pull/17866

If you want, you can test it out before it's released 😊  testing steps are included in the PR description.

Hope that helps! 💪

**Reply by ha1f** [reply]
Thank you for your help!

That looks awesome👏
